opal/mca/btl/usnic/btl_usnic_module.c

```diff
@@ -1611,6 +1611,29 @@ static int create_ep(opal_btl_usnic_module_t* module,
         return OPAL_ERR_OUT_OF_RESOURCE;
     }
 
+    /* Check to ensure that the RX/TX queue lengths are at least as
+       long as we asked for */
+    if (channel->info->rx_attr->size < channel->chan_rd_num) {
+        opal_show_help("help-mpi-btl-usnic.txt",
+                       "internal error during init",
+                       true,
+                       opal_process_info.nodename,
+                       module->linux_device_name,
+                       "endpoint RX queue length is too short", __FILE__, __LINE__,
+                       rc, fi_strerror(-rc));
```

Comment:
Need to set rc and string to some reasonable value, because rc will be 0.  Same for all the others, below.  Maybe FI_ETOOSMALL?